pkg/appfile/appfile.go

```diff
@@ -669,7 +671,7 @@ func generateTerraformConfigurationWorkload(wl *Workload, ns string) (*unstructu
 	}
 
 	configuration := terraformapi.Configuration{
-		TypeMeta: metav1.TypeMeta{APIVersion: "terraform.core.oam.dev/v1beta1", Kind: "Configuration"},
+		TypeMeta: metav1.TypeMeta{APIVersion: "terraform.core.oam.dev/v1beta2", Kind: "Configuration"},
```

Comment:
do we need to do some migrate here?

Reply:
Yes, I did in https://github.com/kubevela/kubevela/pull/3503/files/cde8d3a9577d1049b4325b52427b8ec2c2853653#diff-abd52d53959b7e73bba4c28566a77193d4ac241dc02f1fc17e149dbb8fa4725eL237-R244.